internal/globalMerge.ecl

```diff
@@ -138,7 +138,7 @@ EXPORT globalMerge := MODULE
             if(rc < pc)
             {
               row = rowBuilder.ensureCapacity(returnsize, NULL);
-              void * pos = row;
+              uint8_t * pos = row;
```

Comment:
I think if you rebase your branch with master, then this change/file will disappear from this PR.